src/component/handlers/edit/commands/keyCommandBackspaceToStartOfLine.js

```diff
@@ -24,18 +25,24 @@ function keyCommandBackspaceToStartOfLine(
   var afterRemoval = removeTextWithStrategy(
     editorState,
     strategyState => {
+      var selection;
       var domSelection = global.getSelection();
       var range = domSelection.getRangeAt(0);
-      range = expandRangeToStartOfLine(range);
-
-      var selection = getDraftEditorSelectionWithNodes(
-        strategyState,
-        null,
-        range.endContainer,
-        range.endOffset,
-        range.startContainer,
-        range.startOffset
-      ).selectionState;
+
+      if (range.collapsed && range.startOffset === 0) {
```

Comment:
Gotcha. Using the DOM selection seemed like a really dirty way to do this. Makes a lot more sense.